Downloading (…)okenizer_config.json:   0%|                                 | 0.00/254 [00:00<?, ?B/s]Downloading (…)okenizer_config.json: 100%|██████████████████████████| 254/254 [00:00<00:00, 47.1kB/s]
Downloading (…)lve/main/config.json:   0%|                                 | 0.00/608 [00:00<?, ?B/s]Downloading (…)lve/main/config.json: 100%|███████████████████████████| 608/608 [00:00<00:00, 112kB/s]
Downloading (…)solve/main/vocab.txt:   0%|                                | 0.00/242k [00:00<?, ?B/s]Downloading (…)solve/main/vocab.txt: 100%|█████████████████████████| 242k/242k [00:00<00:00, 476kB/s]Downloading (…)solve/main/vocab.txt: 100%|█████████████████████████| 242k/242k [00:00<00:00, 475kB/s]
Downloading (…)cial_tokens_map.json:   0%|                                 | 0.00/112 [00:00<?, ?B/s]Downloading (…)cial_tokens_map.json: 100%|███████████████████████████| 112/112 [00:00<00:00, 160kB/s]
/root/miniconda3/envs/py38/lib/python3.8/site-packages/huggingface_hub/file_download.py:980: UserWarning: Not enough free disk space to download the file. The expected file size is: 436.70 MB. The target location /root/autodl-tmp/.cache/huggingface/hub only has 0.39 MB free disk space.
  warnings.warn(
/root/miniconda3/envs/py38/lib/python3.8/site-packages/huggingface_hub/file_download.py:980: UserWarning: Not enough free disk space to download the file. The expected file size is: 436.70 MB. The target location /root/autodl-tmp/.cache/huggingface/hub/models--GroNLP--bert-base-dutch-cased/blobs only has 0.39 MB free disk space.
  warnings.warn(
Downloading model.safetensors:   0%|                                      | 0.00/437M [00:00<?, ?B/s]Downloading model.safetensors:   2%|▋                            | 10.5M/437M [00:01<01:15, 5.63MB/s]Traceback (most recent call last):
  File "output/hf-eval-data-v2/f00520_fill_mask_dutch_sentence.py", line 38, in <module>
    test_fill_mask_dutch_sentence()
  File "output/hf-eval-data-v2/f00520_fill_mask_dutch_sentence.py", line 34, in test_fill_mask_dutch_sentence
    assert fill_mask_dutch_sentence(test_sentence) == expected_output
  File "output/hf-eval-data-v2/f00520_fill_mask_dutch_sentence.py", line 18, in fill_mask_dutch_sentence
    model = AutoModel.from_pretrained('GroNLP/bert-base-dutch-cased')
  File "/root/miniconda3/envs/py38/lib/python3.8/site-packages/transformers/models/auto/auto_factory.py", line 563, in from_pretrained
    return model_class.from_pretrained(
  File "/root/miniconda3/envs/py38/lib/python3.8/site-packages/transformers/modeling_utils.py", line 2773, in from_pretrained
    resolved_archive_file = cached_file(pretrained_model_name_or_path, filename, **cached_file_kwargs)
  File "/root/miniconda3/envs/py38/lib/python3.8/site-packages/transformers/utils/hub.py", line 429, in cached_file
    resolved_file = hf_hub_download(
  File "/root/miniconda3/envs/py38/lib/python3.8/site-packages/huggingface_hub/utils/_validators.py", line 118, in _inner_fn
    return fn(*args, **kwargs)
  File "/root/miniconda3/envs/py38/lib/python3.8/site-packages/huggingface_hub/file_download.py", line 1429, in hf_hub_download
    http_get(
  File "/root/miniconda3/envs/py38/lib/python3.8/site-packages/huggingface_hub/file_download.py", line 554, in http_get
    temp_file.write(chunk)
  File "/root/miniconda3/envs/py38/lib/python3.8/tempfile.py", line 473, in func_wrapper
    return func(*args, **kwargs)
OSError: [Errno 28] No space left on device
Downloading model.safetensors:   2%|▋                            | 10.5M/437M [00:01<01:19, 5.34MB/s]